test/parallel/test-worker-terminate-http2-respond-with-file.js

```diff
@@ -33,7 +33,7 @@ if (isMainThread) {
     assert.strictEqual(headers[':status'], 200);
   }));
 
-  req.on('data', common.mustCall(process.exit));
+  req.on('data', common.mustCall(() => process.exit()));
```

Comment:
This doesn't have to be changed, right?

Reply:
Seemingly it's needed since a `Buffer` instance is passed for the listener.
```js
req.on('data', common.mustCall((chunk) => process.exit()));
```